### PR TITLE
Closure script revised with simpler example

### PR DESCRIPTION
**Pull request by avishekjana**

### CODE REVIEW

File: nodejs-deep-dive/closures-in-js/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+# Text
+In this lesson, you will learn a very key concept of JavaScript, the **Closures**. 
+I will explain in clear terms what **closure** is, and how you can easily relate it to real-world.
```
Comment (mjzac) [inline nodejs-deep-dive/closures-in-js/README.md:3]:
```suggestion
```

File: nodejs-deep-dive/closures-in-js/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+I guess you haven't heard of this term *closure* previously, and frankly it is not very much self-explanatory as well. From today onwards, you can remember it with a fairly simple term (or synonym) - Legacy! Why, let's find out.
```
Comment (mjzac) [inline nodejs-deep-dive/closures-in-js/README.md:5]:
```suggestion
You probably haven't heard of the term *closure* previously, and it is not very self-explanatory either. From today onwards, you can remember it with a reasonably simple term (or synonym) - Legacy! Why? Let's find out.
```

File: nodejs-deep-dive/closures-in-js/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    function child() {   
+        var mobile = 'iPhone 13'; 
+        console.log('I have: ', house, mobile);
+    }
+   
+    return child;
+}
+var legacy = parent();
+console.log(typeof(legacy)); //legacy is of type function
+legacy(); // Output: I have: Home Sweet Home iPhone 13
```
Comment (mjzac) [inline nodejs-deep-dive/closures-in-js/README.md:30]: The example seems contrived.. I know it is a hard topic to write a simple example. But wouldn't a greet function be enough?

```js
function generateGreetings(name) {
  function spanish() {
    console.log(`Hola ${name}!`);
  }
  function english() {
    console.log(`Hello ${name}!`);
  }
  return {spanish, english};
};

const name = 'John';
const greetings = generateGreetings(name);

console.log(typeof(greetings.spanish)); // function

greetings.spanish(); // Hola John!
greetings.english(); // Hello John!
```

**Review by mjzac — CHANGES_REQUESTED** [review]
The explanations would become much simpler by saying  parent value is captured and stored till the child go out of scope or destroyed.

**Comment by avishekjana** [comment]
@mjzac I've updated the script as per your review. Please check once.

**Review by mjzac — APPROVED** [review]
Looks good to me.

**Pull request merged**